README.md

```diff
@@ -26,6 +26,7 @@
 |11|Integrate graph toolkit component in teams tab      | Microsoft Teams tab sample app for demonstrating graph toolkit component |                      | [View][tab-graph-toolkit#js]|
 |12|Device permissions      | Microsoft Teams tab sample app for demonstrating device permissions |                      | [View][tab-device-permissions#js]|
 |13|Show adaptive cards in Teams tab | Microsoft Teams tab sample code which demonstrates how to [show adaptive cards in Teams tab](https://docs.microsoft.com/microsoftteams/platform/tabs/how-to/build-adaptive-card-tabs) | [View][tab-adaptive-cards#csharp]               |                | [View][tab-adaptive-cards#js]|
+|13|Tab in stage view      | Microsoft Teams tab sample app for demonstrating stage view in tab | [View][tab-stage-view#csharp]|                 |
```

Comment:
Rephrase it as: Microsoft Teams tab sample app for demonstrating tab in stage view

Reply:
Rephrased